modules/tinymce/src/plugins/link/test/ts/browser/DialogOpenTest.ts

```diff
@@ -69,18 +69,69 @@ describe('browser.tinymce.plugins.link.DialogOpenTest', () => {
     await pDialogCheck(editor);
   });
 
-  it('TINY-7993: Dialog shows correctly right after inserting a link', async () => {
-    const editor = hook.editor();
-    editor.setContent('<p>aaa bbb ccc</p>');
-    TinySelections.setSelection(editor, [ 0, 0 ], 'aaa '.length, [ 0, 0 ], 'aaa bbb'.length);
-    editor.execCommand('mceLink');
-    await TinyUiActions.pWaitForDialog(editor);
-    FocusTools.setActiveValue(SugarDocument.getDocument(), 'http://tiny.cloud');
-    TinyUiActions.submitDialog(editor);
-    await Waiter.pTryUntil('Wait for content to change', () =>
-      TinyAssertions.assertContent(editor, '<p>aaa <a href="http://tiny.cloud">bbb</a> ccc</p>'));
-    editor.execCommand('mceLink');
-    await TinyUiActions.pWaitForDialog(editor);
-    assert.equal(FocusTools.getActiveValue(SugarDocument.getDocument()), 'http://tiny.cloud', 'Should be equal to link\'s href');
-  });
+  const testDialogOnSelection = (
+    startPath: number[],
+    soffset: number,
+    finishPath: number[],
+    foffset: number,
+    result: string,
+    message: string
+  ) => {
+    it(`TINY-7993: Dialog shows correctly right after inserting a link: ${message}`, async () => {
```

Comment:
Don't put `it(...)` call in the function body. It is hard to track down what test exactly failed. Instead, wrap every `testDialgOnSelection` call within `it()`

Reply:
But it has a distinct message for each `it` block. It should be quite obvious which test has failed, no?

Reply:
I meant exploring the error stack  on line numbers in module